README.md

```diff
@@ -52,9 +52,9 @@ ansible-galaxy install kazauwa.headscale
 - `headscale_config_template`
   - Default: `""`
   - Description: path to Jinja2 formatted headscale config template. If present, will override `headscale_config`.
-- `headscale_acl`
-  - Default: `{}`
-  - Description: yaml formatted ACL policies. **Make sure** that you've read the [docs](https://github.com/juanfont/headscale/tree/main/docs#policy-acls) on how to use this feature.
+- `headscale_acl_file`
+  - Default: `""`
+  - Description: path to HuJSON formatted headscale acl file.
```

Comment:
I think a link to ACL syntax here would be helpful for a reader: https://tailscale.com/kb/1337/acl-syntax